.changeset/olive-seals-give.md

```diff
@@ -0,0 +1,8 @@
+---
+'@guardian/identity-auth': minor
+---
+
+- Add `decodeTokens` method to `Token` class
+- Refactor library so that we only store required values in l `verifyTokens`ocal storage, and now derive other values from decoding the token
```

Comment:
```suggestion
- Refactor library so that we only store required values in `verifyTokens` local storage, and now derive other values from decoding the token
```